app/controllers/npq_separation/admin/statements_controller.rb

```diff
@@ -0,0 +1,84 @@
+class NpqSeparation::Admin::StatementsController < NpqSeparation::AdminController
```

Comment:
How about inheriting from `SuperAdminController`?

Reply:
Hmm, maybe split the super admin checks into a module and share between classes, I'm keen for us not to be repeating auth code - can't see a problem with whats there _but_ its better to have a single implementation